evm_arithmetization/src/cpu/kernel/asm/mpt/hash/hash.asm

```diff
@@ -1,4 +1,5 @@
-// Computes the Merkle root of the given trie node.
+// Computes the Merkle root of the given trie node. RLP encoding (if any) will
```

Comment:
`rlp_start` here is a value already on the stack, so could be arbitrary (from a local perspective), hence the addition in the comment feels superfluous to me